### DISCUSSION TITLE
Amiberry Dietpi ARMv8 64-bit image installation

### DISCUSSION THREAD

**Discussion by XxLOFDxBERLINxX** [BlitterStudio/amiberry#903] [Q&A] [answered]
Hello folks, is there a way to manually install Amiberry on Dietpi ARMv8 64-bit image ?

amiberry-v4.1.6-rpi4-sdl2-64bit-rpios.zip

would be very grateful for a solution

**Comment by midwan** [comment, marked as answer]
You can always install it from source, using a relevant target (either the 64-bit DISPMANX or SDL2). The current list of supported platforms is listed in the wiki: https://github.com/midwan/amiberry/wiki/Available-Platforms

I believe the DietPi maintainers were looking into adding new platform options however, based on a recent comment I saw. So perhaps you can also ping them to request they add it in their installer for ARMv8 64-bit images directly, which would make things a bit easier.
